enslig-forsorger/src/main/kotlin/no/nav/familie/kontrakter/ef/iverksett/IverksettDto.kt

```diff
@@ -76,6 +80,18 @@ data class VedtaksperiodeDto(
         val periodeType: VedtaksperiodeType
 )
 
+data class FeilutbetalingDto(
+        val tilbakekrevingsvalg: Tilbakekrevingsvalg,
+        // Kreves bare hvis tilbakekrevingsvalg = OPPRETT_TILBAKEKREVING_MED_VARSEL
+        val varseltekst: String?,
+        // Kreves bare hvis tilbakekrevingsvalg = OPPRETT_TILBAKEKREVING_MED_VARSEL.
+        // Hvis det er påkrevd og mangler, så vil det hentes fra simulering
+        val sumFeilutbetaling: BigDecimal?,
+        // Kreves bare hvis tilbakekrevingsvalg = OPPRETT_TILBAKEKREVING_MED_VARSEL.
+        // Hvis det er påkrevd og mangler, så vil det hentes fra simulering
+        val perioder: List<Periode>?
```

Comment:
Er det sånn att alle disse er obligatoriske når vi har tilbakereving med varsel? 
Burde vi legge disse i et eget objekt då de bare , eks 
```
data class FeilutbetalingDto(
        val tilbakekrevingsvalg: Tilbakekrevingsvalg,
        // Kreves bare hvis tilbakekrevingsvalg = OPPRETT_TILBAKEKREVING_MED_VARSEL
        val tilbakekrevingMedVarsel: TilbakekrevingMedVarselDto?
)

data class TilbakekrevingMedVarselDto(val varseltekst: String,
                                      val sumFeilutbetaling: BigDecimal,
                                      val perioder: List<Periode>)
```

Reply:
Jeg liker den! Men hvis vi skal åpne for at `sumFeilutbetalinger` og `perioder` skal beregnes på iverksett-siden, så må de fortsatt kunne være null når de sendes over.